feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss

```diff
@@ -23,6 +23,9 @@
 
   .cx-unit-level-order-history-filter-label-wrapper {
     width: 200px;
+    @include forFeature('a11yTruncatedTextUnitLevelOrderHistory') {
+      width: unset;
```

Comment:
Since the styles for [`text-ellipsis`](https://github.com/SAP/spartacus/blob/4d970ec9ca2225b6450b7039b7ab8fa94def1b04/feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss#L113-L118) class will not be used after the FF is removed let's mark them with a comment.

Maybe sth like: `// TODO: Remove following block once 'a11yTruncatedTextUnitLevelOrderHistory' feature flag is removed`